diff --git a/experiments/run_our_method.py b/experiments/run_our_method.py
index 244bd1b..118004e 100644
--- a/experiments/run_our_method.py
+++ b/experiments/run_our_method.py
@@ -264,7 +264,7 @@ def run_postfiltering_experiment(all_results, dataset_name, filter_width, alpha)
     data, queries, filter_values, metric = initialize_dataset(dataset_name)
 
     build_params = wp.BuildParams(
-        64, 500, alpha, f"index_cache/{dataset_name}/unsorted-"
+        32, 80, alpha, f"index_cache/{dataset_name}/unsorted-"
     )
     postfilter_constructor = wp.postfilter_vamana_constructor(metric, "float")
     postfilter_build_start = time.time()
@@ -306,7 +306,7 @@ def get_vamana_tree(data, filter_values, metric, alpha, split_factor):
     vamana_tree_constructor = wp.vamana_range_filter_tree_constructor(metric, "float")
 
     vamana_tree_build_start = time.time()
-    build_params = wp.BuildParams(64, 500, alpha, f"index_cache/{dataset_name}/")
+    build_params = wp.BuildParams(32, 80, alpha, f"index_cache/{dataset_name}/")
     vamana_tree = vamana_tree_constructor(
         data,
         filter_values,
@@ -485,7 +485,7 @@ def run_super_optimized_postfiltering_experiment(
     start_memory = resource.getrusage(resource.RUSAGE_SELF).ru_maxrss
 
     build_params = wp.BuildParams(
-        64, 500, alpha, f"index_cache/{dataset_name}-super_opt_postfiltering/"
+        32, 80, alpha, f"index_cache/{dataset_name}-super_opt_postfiltering/"
     )
     super_optimized_postfilter_tree = constructor(
         data,
diff --git a/experiments/wrapper.py b/experiments/wrapper.py
index 312d2e3..bf27499 100644
--- a/experiments/wrapper.py
+++ b/experiments/wrapper.py
@@ -242,7 +242,7 @@ def range_filter_tree_index_constructor(metric, dtype):
 def prefilter_index_constructor(metric, dtype):
     if metric == "Euclidian":
         if dtype == "uint8":
-            return PrefilterIndexUint8Euclidian
+            return PrefilterIndexUInt8Euclidian
         elif dtype == "int8":
             return PrefilterIndexInt8Euclidian
         elif dtype == "float":
@@ -251,7 +251,7 @@ def prefilter_index_constructor(metric, dtype):
             raise Exception("Invalid data type " + dtype)
     elif metric == "mips":
         if dtype == "uint8":
-            return PrefilterIndexUint8Mips
+            return PrefilterIndexUInt8Mips
         elif dtype == "int8":
             return PrefilterIndexInt8Mips
         elif dtype == "float":
@@ -265,7 +265,7 @@ def prefilter_index_constructor(metric, dtype):
 def postfilter_vamana_constructor(metric, dtype):
     if metric == "Euclidian":
         if dtype == "uint8":
-            return PostfilterVamanaIndexUint8Euclidian
+            return PostfilterVamanaIndexUInt8Euclidian
         elif dtype == "int8":
             return PostfilterVamanaIndexInt8Euclidian
         elif dtype == "float":
@@ -274,7 +274,7 @@ def postfilter_vamana_constructor(metric, dtype):
             raise Exception("Invalid data type " + dtype)
     elif metric == "mips":
         if dtype == "uint8":
-            return PostfilterVamanaIndexUint8Mips
+            return PostfilterVamanaIndexUInt8Mips
         elif dtype == "int8":
             return PostfilterVamanaIndexInt8Mips
         elif dtype == "float":
@@ -288,7 +288,7 @@ def postfilter_vamana_constructor(metric, dtype):
 def vamana_range_filter_tree_constructor(metric, dtype):
     if metric == "Euclidian":
         if dtype == "uint8":
-            return VamanaRangeFilterTreeIndexUint8Euclidian
+            return VamanaRangeFilterTreeIndexUInt8Euclidian
         elif dtype == "int8":
             return VamanaRangeFilterTreeIndexInt8Euclidian
         elif dtype == "float":
@@ -297,7 +297,7 @@ def vamana_range_filter_tree_constructor(metric, dtype):
             raise Exception("Invalid data type " + dtype)
     elif metric == "mips":
         if dtype == "uint8":
-            return VamanaRangeFilterTreeIndexUint8Mips
+            return VamanaRangeFilterTreeIndexUInt8Mips
         elif dtype == "int8":
             return VamanaRangeFilterTreeIndexInt8Mips
         elif dtype == "float":
@@ -311,7 +311,7 @@ def vamana_range_filter_tree_constructor(metric, dtype):
 def super_optimized_postfilter_tree_constructor(metric, dtype):
     if metric == "Euclidian":
         if dtype == "uint8":
-            return SuperOptimizedPostfilterTreeIndexUint8Euclidian
+            return SuperOptimizedPostfilterTreeIndexUInt8Euclidian
         elif dtype == "int8":
             return SuperOptimizedPostfilterTreeIndexInt8Euclidian
         elif dtype == "float":
@@ -320,7 +320,7 @@ def super_optimized_postfilter_tree_constructor(metric, dtype):
             raise Exception("Invalid data type " + dtype)
     elif metric == "mips":
         if dtype == "uint8":
-            return SuperOptimizedPostfilterTreeIndexUint8Mips
+            return SuperOptimizedPostfilterTreeIndexUInt8Mips
         elif dtype == "int8":
             return SuperOptimizedPostfilterTreeIndexInt8Mips
         elif dtype == "float":
